src/pyfaf/actions/archadd.py

```diff
@@ -16,6 +16,8 @@
 # You should have received a copy of the GNU General Public License
 # along with faf.  If not, see <http://www.gnu.org/licenses/>.
 
+from wtforms import validators
```

Comment:
I really don't like pulling wtforms inside the CLI. Is there not a way around that?

Reply:
Fixed a mildly ugly workaround (using `exec()`) to pass the validators as strings. The alternative would probably be to write a separate method for each validator and for each of the two parsers.

Reply:
```
from wtforms import validators

for validator in kwargs.get('validators'):
    class = getattr(validators, validator)
    field_kwargs['validators'].append(class())
```
?

Reply:
Of course, you would still need to add WTForms validator class names from the CLI.

Reply:
Thanks! How silly of me not to think of that.